<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>진단 보조 시스템</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDialogButtonBox" name="CheckBox">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>52</y>
      <width>156</width>
      <height>31</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Apply|QDialogButtonBox::Help</set>
    </property>
    <property name="centerButtons">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="DiseaseEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>761</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="DiseaseList">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>761</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="AnalyzeBox">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>130</y>
      <width>156</width>
      <height>23</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Apply|QDialogButtonBox::Reset</set>
    </property>
   </widget>
   <widget class="QLabel" name="DiseaseListLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>현재 추가된 병 리스트</string>
    </property>
   </widget>
   <widget class="QLabel" name="AddDisease">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>추가할 병명</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>986</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
